config/config_test.go

```diff
@@ -605,6 +607,49 @@ var expectedConf = &Config{
 				},
 			},
 		},
+		{
+			JobName: "digitalocean-droplets",
+
+			HonorTimestamps: true,
+			ScrapeInterval:  model.Duration(15 * time.Second),
+			ScrapeTimeout:   DefaultGlobalConfig.ScrapeTimeout,
+
+			MetricsPath: DefaultScrapeConfig.MetricsPath,
+			Scheme:      DefaultScrapeConfig.Scheme,
+
+			ServiceDiscoveryConfig: sd_config.ServiceDiscoveryConfig{
+				DigitalOceanSDConfigs: []*digitalocean.SDConfig{
+					{
+						HTTPClientConfig: config_util.HTTPClientConfig{
+							BearerToken: "abcdef",
```

Comment:
HTTPClientConfig is used in do_sd. Is it necessary to update some of the `resolveFilepaths` functions in `config` pkg.

Reply:
This is done in the release branch and will be merged in master